Chartboost/ChartboostRouter.m

```diff
@@ -10,6 +10,7 @@
 
 static NSString * const kChartboostAppIdKey        = @"appId";
 static NSString * const kChartboostAppSignatureKey = @"appSignature";
+static NSString * const kGADMAdapterMinimumOSVersion = @"10.0";
```

Comment:
Should we use a different name for MoPub? GADM Are the GoogleMobileAds prefix.
kChartboostMinimumOSVersion?

Reply:
👍